wordcount_and_median.py

```diff
@@ -1,41 +1,40 @@
 import os
-from collections import Counter
 import re
+import csv
+import sys
 import bisect
-from statistics import median
-
+import statistics
+from collections import Counter
 
 
-def wc_m_main(input_folder = 'wc_input', output_folder = 'wc_output',\
-              med_out_file = 'med_result.txt', wc_out_file = 'wc_result.txt'):
-    linelengths, fulldict, files_list = __initializer(input_folder)
-    linelengths, fulldict = file_reader(linelengths, fulldict,\
+def wc_m_main(input_folder='wc_input', output_folder='wc_output',
+              med_out_file='med_result.txt', wc_out_file='wc_result.txt'):
+    linelengths, fulldict, files_list = _initializer(input_folder)
+    linelengths, fulldict = file_reader(linelengths, fulldict,
                                         files_list, input_folder)
     count_writer(output_folder, wc_out_file, fulldict)
     median_writer(output_folder, med_out_file, linelengths)
 
 
-
-def __initializer(input_folder_i):
-    line_length_list = [];
-    dictionary_initializer = Counter();
-    input_files_list = os.listdir(input_folder_i)
-    input_files_list.sort()
+def _initializer(input_folder_i):
+    line_length_list = []
+    dictionary_initializer = Counter()
+    input_files_list = sorted(os.listdir(input_folder_i))
     return line_length_list, dictionary_initializer, input_files_list
 
 
 def file_reader(linelengths_fr, fulldict_fr, files_list_fr, input_folder_fr):
     for file in files_list_fr:
-        with open(input_folder_fr+'/'+file, 'r') as f:
+        with open(os.path.join(input_folder_fr, file), 'r') as f:
```

Comment:
``os.path.join`` is a function which will always use the correct directory character (e.g. ``/`` on Linux, ``\`` on Windows, etc).